apps/tsilhqotin-language-hub/src/Components/Pages/SongDetail/SongDetail.tsx

```diff
@@ -0,0 +1,16 @@
+import './SongDetail.module.css';
+
+/* eslint-disable-next-line */
+export interface SongDetailProps {}
+
+export function SongDetail(props: SongDetailProps) {
+    return (
+        <div>
+            {/*
+            <Songs songData={componentState.songData}></Songs>
```

Comment:
Finish implementing. If this is just a presentational component, the `SongDetailProps` should essentially be a `SongViewModel`-i.e. all the data that is on a single song instance that comes from the API.

Reply:
I renamed SongDetailProps to SongViewModel.